frontend/src/components/Issue/IssueBanner.vue

```diff
@@ -39,4 +49,23 @@ const showPendingApproval = computed(() => {
   const task = activeTask(issue.value.pipeline);
   return task.status == "PENDING_APPROVAL";
 });
+
+const showEarliestAllowedTimeBanner = computed(() => {
+  const task = activeTask(issue.value.pipeline);
```

Comment:
What if there are multiple pending tasks? It will always show the active one.
I was thinking of showing the banner for the selected task.